docs/scripts/check_swatinit.rst

```diff
@@ -0,0 +1,177 @@
+* A text table printed to the terminal/stdout summarizing volumetric changes
+  from SWATINIT to SWAT by QC category. This can also be plotted as a
+  waterfall chart.
+* A text table with maximum capillary pressure pr. input SATNUM
+* A text table with maximum capillary pressure scaling pr. EQLNUM and SATNUM.
+* A plot panel with scatter plot of a parameter vs depth for every cell. This
+  plot panel is individual pr. EQLNUM.
+
+The Eclipse keyword SWATINIT
+----------------------------
+
+SWATINIT is used as a tool to conserve the water saturation modelling done
+in the geomodel. However, SWATINIT will not set the "initial" water saturation
+in the Eclipse run, it will *only* scale the capillary pressure table
+on a cell by cell basis, with *exceptions*. This tool will assess the impact
+of these exceptions.
+
+Water saturation initialization in Eclipse at time zero is *always* performed
+using the capillary pressure function from the SWOF/SWFN tables, this is in
+order to maintain initial dynamical stability. The impact of SWATINIT is only
+to allow for cell-by-cell scaling of the capillary pressure function before it
+is used for initialization.
+
+.. figure:: images/ecl-swat-initialization.png
+   :align: center
+   :width: 90%
+
+   Illustration of the water initialization in Eclipse. If SWATINIT is in use
+   the capillary pressure curve to the right is scaled vertically up or down
+   such that the computed SWAT matches the requested SWATINIT value for the
+   specific cell.
+
+Cell by cell outcome of water initialization
+--------------------------------------------
+
+The tool assesses each cell in the dynamical model individually, and flags
+them according to what has happened from SWATINIT to SWAT. The flag will be
+included in the ``QC_FLAG`` column of the outputted CSV. The possible outcomes
+are:
+
+``PC_SCALED``
+  Capillary pressure have been scaled and SWATINIT was accepted. Zero
+  volumetric change, but check the maximum capillary pressure pr SATNUM in each
+  EQLNUM to ensure extreme values were not necessary.
+
+``WATER``
+  SWATINIT was 1 in the water zone, and SWAT is set to 1. All good.
+
+``SWL_TRUNC``
+  If SWL, as given to Eclipse through SWOF or through the SWL keyword, is larger
+  than SWATINIT, SWAT will be reset to SWL. Compared to SWATINIT, extra water
+  is added to the model and hydrocarbons are lost. If this amounts to
+  significant volumes, revise the modelling.
+
+``SWATINIT_1``
+  When SWATINIT is 1 above the contact, Eclipse will ignore SWATINIT in the cell
+  and not touch the capillary pressure function. This will typically result in extra
+  hydrocarbons added to the model for a normal capillary pressure function. This
+  could be ok as long as the porosities and/or permeabilities of these cells are
+  small. If it is not, you should look into if there are upscaling issues for
+  this cell. In situations with nonzero item #9 in EQUIL, this can also occur
+  below the contact.
```

Comment:
Could mention that having Pc(Sw=1)=epsilon>0 allows Eclipse to match SWATINIT=1 also above the contact.

Reply:
Yes, I think this falls into the PC_SCALED category, with "scaling" to 1.